packages/callhierarchy/src/browser/callhierarchy-context.ts

```diff
@@ -46,6 +46,7 @@ export class CallHierarchyContext implements Disposable {
         return symbols;
     }
 
+    // tslint:disable-next-line:typedef
```

Comment:
`monaco.editor.IReference<MonacoEditorModel>`  or there is some issue with it?

Reply:
I tried previously and got the error:
`Namespace 'monaco.editor' has no exported member 'IReference'.`

```ts
 async getEditorModelReference(uri: string): Promise<monaco.editor.IReference<MonacoEditorModel>> { }
```